frontend/test/acceptance/JoinPartnerSpec.scala

```diff
@@ -59,6 +59,12 @@ class JoinPartnerSpec extends FeatureSpec with Browser
       And("should be logged in with their Identity account.")
       assert(enterDetails.userIsSignedIn)
 
+      When("users select non-UK country,")
+      enterDetails.changeCountry()
+
+      Then("the currency should change.")
```

Comment:
I guess the default currency is GBP, so the currency _will_ have changed? We don't seem to be specifying a `countryGroup` when we go to `/join/supporter/enter-details` - so I'm not sure if there's any chance the currency might already be USD, before the country is set to US.